app/views/mail/emailBaseTemplate.scala.html

```diff
@@ -0,0 +1,53 @@
+@()(content:Html)
+<html>
+
+<body>
+  <style type="text/css">
+    body,
+    p,
+    li,
+    h2 {
+      font-family: "Monospaced Number", -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto,
+        "PingFang SC", "Hiragino Sans GB", "Microsoft YaHei", "Helvetica Neue", Helvetica, Arial,
+        sans-serif;
+      font-size: 14px;
+      line-height: 1.5;
+    }
+  </style>
+  <div style="margin-left: auto; margin-right: auto; max-width: 600px;">
+    <p>
+      <a href="https://webknossos.org"><img src="https://static.webknossos.org/mails/email-header.600.png"
+          alt="webKnossos" style="width: 100%; max-width: 600px;" /></a>
+    </p>
+
+    @content
+
+    <p style="text-align: center; padding-top: 30px;">
+      <a href="https://webknossos.org"><img src="https://static.webknossos.org/mails/icon-webknossos.png"
+          alt="webKnossos" style="width: 50px; height: 50px;" /></a>
+      &nbsp;
+      <a href="https://docs.webknossos.org"><img src="https://static.webknossos.org/mails/icon-docs.png"
+          alt="Documentation" style="width: 50px; height: 50px;" /></a>
+      &nbsp;
+      <a href="https://www.youtube.com/channel/UCVQqjQ50lJORCZuLxnEE-tA"><img
+          src="https://static.webknossos.org/mails/icon-video.png" alt="YouTube Channel"
+          style="width: 50px; height: 50px;" /></a>
+      &nbsp;
+      <a href="https://twitter.com/webknossos"><img src="https://static.webknossos.org/mails/icon-twitter.png"
+          alt="Twitter Channel" style="width: 50px; height: 50px;" /></a>
+    </p>
+
+    <div style="height: 60px;">&nbsp;</div>
+    <p style="color: #888; font-size: 0.6rem;">
+      You are receiving this email because you signed up for an account on webKnossos. You may
+      cancel your account at any time.
```

Comment:
More of a sidenote, but how can users cancel their account? They would need to contact the respective webknossos instance owner?

Reply:
Yes. I noticed that as well. ATM, that means writing an email.